xdrip-get-entries.sh

```diff
@@ -1228,7 +1228,7 @@ function check_recent_sensor_insert()
 
 function  post-nightscout-with-backfill()
 {
-  if [ "$mode" == "Stopped" ]; then
+  if [ "$state" == "Stopped" ]; then
```

Comment:
add check for mode != expired again here I think.

Reply:
I am wondering if we should have this check here at all as well...  as long as there is an entry-backfill.json to be processed... does it matter that the sensor is stopped?